src/Components/Components/src/Routing/RouteContext.cs

```diff
@@ -12,9 +12,7 @@ internal class RouteContext
 
         public RouteContext(string path)
         {
-            // This is a simplification. We are assuming there are no paths like /a//b/. A proper routing
-            // implementation would be more sophisticated.
-            Segments = path.Trim('/').Split(Separator, StringSplitOptions.RemoveEmptyEntries);
+            Segments = path.TrimStart('/').Split(Separator);
```

Comment:
Why change this?

Reply:
Good call -- this is another place where I should add inline comments to clarify the change. I will update the code comment in the source with an explanation.